Makefile

```diff
@@ -184,6 +184,7 @@ RECORD_SHARED_JS_DEPS=$(JS)/vendor/jquery-latest.min.js \
 	$(COMMON)/alerts.js \
 	$(COMMON)/authen.js \
 	$(COMMON)/errors.js \
+	$(COMMON)/filters.js \
```

Comment:
Removing this gives me errors now because alerts.js requires it for the alerts directive.